docs/python-api.rst

```diff
@@ -26,9 +26,9 @@ sequences, for example using :func:`tskit.load`.
 The ``TreeSequence`` class
 ++++++++++++++++++++++++++
 
-.. autoclass:: tskit.TreeSequence()
```

Comment:
I think the brackets were deliberate here because it hides the messy low-level details of how these objects are constructed from the user (who doesn't need to know about this). Is this no longer possible in sphinx 3?

Reply:
Ah ok, I get it. Reverted to brackets.